src/Migration/Reader/Map.php

```diff
@@ -211,16 +210,15 @@ public function isFieldMapped($document, $field, $type)
      */
     public function getDocumentMap($document, $type)
     {
-        $this->validateType($type);
```

Comment:
Validation can be omitted because it will be performed in the `isDocumentIgnored()` method